src/Toml/Parser.hs

```diff
@@ -88,6 +87,17 @@ tableNameP = lexeme $ between (char '[') (char ']') keyP
 
 -- Values
 
+integerP :: Parser Integer
+integerP = lexeme $ binary <|> octal <|> hexadecimal <|> decimal
```

Comment:
Since we don't have unit tests now it's okay to rely only on property-based tests now. So we can be sure that at least we didn't break anything.